.github/workflows/prepare_release.yml

```diff
@@ -0,0 +1,55 @@
+name: Prepare Release
+on:
+  workflow_dispatch:
+  schedule:
+    # Every Friday at 06:15 AM UTC
+    - cron: "15 6 * * 5"
```

Comment:
I would prefer bi-weekly for the beginning. Like 6:00AM every second friday. We can easily adjust this later. Would that be fine?